agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/RequestChecker.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Includes work from:
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.azure.monitor.opentelemetry.exporter.implementation;
+
+import static com.azure.monitor.opentelemetry.exporter.implementation.AiSemanticAttributes.JOB_SYSTEM;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import java.util.function.Function;
+
+public final class RequestChecker {
+
+  private RequestChecker() {}
+
+  public static boolean isRequest(SpanData span) {
+    return isRequest(span.getKind(), span.getParentSpanContext(), span.getAttributes()::get);
+  }
+
+  public static boolean isRequest(ReadableSpan span) {
```

Comment:
Not related specifically to this PR.  I just noticed there aren't any documentation comments for methods.  Wondering why!

Reply:
My understanding is that we document the methods that our users may use. This one is not susceptible to being used by our users. An example [here](https://github.com/microsoft/ApplicationInsights-Java/blob/568a2373dc479181b214a7278aeb1dabe35e5a32/agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java#L34) of a method that our users can use.